Firestore/core/test/firebase/firestore/model/field_value_test.cc

```diff
@@ -38,6 +43,125 @@ const uint8_t* Bytes(const char* value) {
 
 }  // namespace
 
+TEST(FieldValueTest, ExtractsFields) {
```

Comment:
The other tests in this file are named 'FieldValue' rather than 'FieldValueTest'. (I think most of our other tests omit the trailing 'Test' in this case too.)

Reply:
We've discussed this before, though I couldn't find it.

The trouble with `TEST(FieldValue, ExtractsFields)` is that it works right up until you want to create a test fixture. At that point you'd need to create a class `FieldValue` to represent the fixture but that would collide with the class we're testing here. Better to just call these all FooTest so that you don't need to rename them all later.

I had been slowly migrating existing usages in this direction (but didn't completely move this file, mostly to avoid conflicts with later commits).

Reply:
Ah, right. Until everything's consistent, you'll probably have to keep reminding me of this. :)